### PR TITLE
Add radio, os, speech to help() in the REPL

### PR DESCRIPTION
**Pull request by bfayers**

### CODE REVIEW

**Comment by dpgeorge** [comment]
Please can you put them in alphabetical order in the existing list, and make sure there is a comma at the end of the line (where necessary).  Thanks!

**Comment by bfayers** [comment]
Done this, didn't think about ordering earlier, oops.

File: source/microbit/help.c [hunk 1]
```diff
@@ -54,7 +54,8 @@ STATIC const char *help_text =
 "  CTRL-E        -- enter paste mode, turning off auto-indent\n"
 "\n"
 "Available modules: antigravity, array, collections, gc, love, math,\n"
-"micropython, music, neopixel, random, struct, sys, this\n"
+"micropython, music, neopixel, os, radio, random,\n" 
+"speech, struct, sys, this,\n"
```
Comment (dpgeorge) [inline source/microbit/help.c]: Sorry to be a pain, but can you please make these 2 new lines into 1 line, with all the modules listed from "micropython" to "this"?  They will fit within 80 chars and it's best to reduce the total number of lines.  Also, the end of the line should be left without a comma or period, just a newline like it was previously.  Thanks!

**Comment by bfayers** [comment]
Thats fine, Done. :)

**Comment by dpgeorge** [comment]
Ok, final thing: please squash your 3 commits into 1, eg: `git rebase -i master`.

**Comment by bfayers** [comment]
Sorry, new to this whole git thing. I was editing in the browser...

**Pull request closed without merging**